frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/templates/kubevirt-template.hbs

```diff
@@ -181,5 +194,4 @@ spec:
   searchCollector:
     enabled: true
   certPolicyController:
-    enabled: true
-{{/if}}
\ No newline at end of file
+    enabled: true{{/if}}
```

Comment:
Is there a reason for this change? If not, please put the closing `{{/if}}` on a newline, and end the file with another newline.